hack/jenkins/release_build_and_upload.sh

```diff
@@ -31,6 +31,11 @@ export DEB_VERSION=${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_BUILD}
 export RPM_VERSION=${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_BUILD}
 export GOPATH=~/go
 
+# Make sure the right golang version is installed based on Makefile
+EXPCECTED_GOLANG_VERSION=$(cat Makefile | grep "GO_VERSION ?=" | awk -F"= " '{print $2}')
+source ./hack/jenkins/installers/check_install_golang.sh $EXPCECTED_GOLANG_VERSION
```

Comment:
Omit the `source` keyword. In general, scripts should run in their own subshell to avoid namespace pollution.

Reply:
I had that thought when writing it but forgot to listen to my thought :D